CMSgov/cms-carts-seds
# Mdct 238 -  serverless 3

# Description
Related Ticket: https://qmacbis.atlassian.net/browse/MDCT-238
Upgrate carts to serverless 3.

You're going to need to copy any new values in the example env into your .env in order to run

## How to test
Run it local, check that the deployed branch is functional, and that the deploy seems sane.

## Dependencies

You're going to need to copy any new values in the example env into your .env in order to run

# Get it done
Still need to get an updated version of the cms bucket-helper in here, that can either be a follow-on, or we can flag the ticket until that is updated.

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
